CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ the code was deployed.
 
 ## [Unreleased]
 
+## [13.9.0] - 2024-05-08
+
+### Added
+
+- Configured buttons for RAK7201v2, now version 2 buttons will send out alerts.
```

Comment:
Please include the Clickup task ID for the task associated with this PR in the same format as the other entries in the Changelog.
![image](https://github.com/bravetechnologycoop/BraveButtons/assets/144853996/b9d8fce1-f6ec-4b14-bb85-9a8aadfab95f)

Reply:
Ok I added this!